tests/test_models.py

```diff
@@ -228,3 +229,13 @@ def test_new_item():
         new_item.comment == comment and
         new_item.company_id == company_id
     )
+
+def test_new_item_history():
+    """ Test creation of new ItemHistory """
+    emp_id = 2
+    item_id = 1
+    item_history = ItemHistory(employee_id=emp_id, item_id=item_id)
+    assert (
```

Comment:
@ammaratef45 let's do assert for every test:
`assert item_history.employee_id == emp_id`
`assert item_history.item_id == item_id`

Reply:
We should follow the one test, one assertion rule, right?